memphis88/application/test/message-store.js

```diff
@@ -0,0 +1,71 @@
+describe('Message Store', function() {
```

Comment:
in this test suite you are dependent on the order of execution. you don't want to go into that place